src/simplify_algebra.cpp

```diff
@@ -412,6 +412,21 @@ struct find_dot_mul
     }
 };
 
+auto fusable_split(const std::string& name)
```

Comment:
Nit: Please put `slice` in the name of this function -- this function is therefore specific to `slice`. Thanks.